
The ident for Seagate 8GB microdrive is
"SEAGATE", "ST1"
hash 0x76dc4190, 0xcfba9599
manfid 0x0111, 0x0000

Signed-off-by: Marcin Juszkiewicz <openembedded@hrw.one.pl>

 drivers/ide/legacy/ide-cs.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.16/drivers/ide/legacy/ide-cs.c
===================================================================
--- linux-2.6.16.orig/drivers/ide/legacy/ide-cs.c	2006-04-20 12:41:41.000000000 +0200
+++ linux-2.6.16/drivers/ide/legacy/ide-cs.c	2006-04-20 12:45:19.640621552 +0200
@@ -444,6 +444,7 @@
 	PCMCIA_DEVICE_PROD_ID12("PCMCIA", "CD-ROM", 0x281f1c5d, 0x66536591),
 	PCMCIA_DEVICE_PROD_ID12("PCMCIA", "PnPIDE", 0x281f1c5d, 0x0c694728),
 	PCMCIA_DEVICE_PROD_ID12("SHUTTLE TECHNOLOGY LTD.", "PCCARD-IDE/ATAPI Adapter", 0x4a3f0ba0, 0x322560e1),
+	PCMCIA_DEVICE_PROD_ID12("SEAGATE", "ST1", 0x76dc4190, 0xcfba9599),	/* Seagate 8GB microdrive */
 	PCMCIA_DEVICE_PROD_ID12("TOSHIBA", "MK2001MPL", 0xb4585a1a, 0x3489e003),
 	PCMCIA_DEVICE_PROD_ID1("TRANSCEND    512M   ", 0xd0909443),
 	PCMCIA_DEVICE_PROD_ID12("WIT", "IDE16", 0x244e5994, 0x3e232852),


-------------------------------------------------------------------------
With pcmcia-cs -> pcmciautils change some cards which was working with
hostap driver now are bound into orinoco driver.

This patch made them bound into hostap like it was before.

Signed-off-by: Marcin Juszkiewicz <openembedded@hrw.one.pl>

--- linux/drivers/net/wireless/hostap/hostap_cs.c.orig	2006-04-09 12:42:34.944903728 +0200
+++ linux/drivers/net/wireless/hostap/hostap_cs.c	2006-04-09 12:42:37.474519168 +0200
@@ -912,6 +912,9 @@
 	PCMCIA_DEVICE_PROD_ID123(
 		"SMC", "SMC2632W", "Version 01.02",
 		0xc4f8b18b, 0x474a1f2a, 0x4b74baa0),
+	PCMCIA_DEVICE_PROD_ID123(
+		"Pretec", "CompactWLAN Card 802.11b", "2.5",
+		0x1cadd3e5, 0xe697636c, 0x7a5bfcf1),
 	PCMCIA_DEVICE_PROD_ID12("BUFFALO", "WLI-CF-S11G", 
 				0x2decece3, 0x82067c18),
 	PCMCIA_DEVICE_PROD_ID12("Compaq", "WL200_11Mbps_Wireless_PCI_Card",
